bitnami/mariadb-galera/values.yaml

```diff
@@ -522,9 +556,25 @@ tolerations: []
 ##     whenUnsatisfiable: DoNotSchedule
 ##
 topologySpreadConstraints: {}
+## @param lifecycleHooks for the galera container(s) to automate configuration before or after startup
+##
+lifecycleHooks: {}
 ## Enable persistence using Persistent Volume Claims
 ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
 ##
+## Galera container port
+## @param containerPorts.mysql mariadb database container port
+## @param containerPorts.galera galera cluster container port
+## @param containerPorts.galera galera IST container port
+## @param containerPorts.galera galera SST container port
+##
+containerPorts:
+  mysql: 3306
+  # TODO: Container image is not aware if the following container's ports
```

Comment:
This sentence is incomplete, and I think it should be something like: 
`# Note: Container image is not aware of changes in the following container's ports`